pt/convolutional-neural-networks.md

```diff
@@ -0,0 +1,718 @@
+**Convolutional Neural Networks translation**
+
+<br>
+
+**1. Convolutional Neural Networks cheatsheet**
+
+&#10230; Dicas de Redes Neurais Convolucionais
+
+<br>
+
+
+**2. CS 230 - Deep Learning**
+
+&#10230; CS 230 - Aprendizagem profunda
+
+<br>
+
+
+**3. [Overview, Architecture structure]**
+
+&#10230; [Visão geral, Estrutura arquitetural]
+
+<br>
+
+
+**4. [Types of layer, Convolution, Pooling, Fully connected]**
+
+&#10230; [Tipos de camadas, Convolução, Pooling, Totalmente conectada]
+
+<br>
+
+
+**5. [Filter hyperparameters, Dimensions, Stride, Padding]**
+
+&#10230; [Hiperparâmetros de filtro, Dimensões, Passo, Preenchimento]
+
+<br>
+
+
+**6. [Tuning hyperparameters, Parameter compatibility, Model complexity, Receptive field]**
+
+&#10230;[Ajustando hiperparâmetros, Compatibilidade de parâmetros, Complexidade de modelo, Campo receptivo]
+
+<br>
+
+
+**7. [Activation functions, Rectified Linear Unit, Softmax]**
+
+&#10230; [Funções de Ativação, Unidade Linear Retificada, Softmax]
+
+<br>
+
+
+**8. [Object detection, Types of models, Detection, Intersection over Union, Non-max suppression, YOLO, R-CNN]**
+
+&#10230;[Detecção de objetos, Tipos de modelos, Detecção, Intersecção por União, Supressão não-máxima, YOLO, R-CNN]
+
+<br>
+
+
+**9. [Face verification/recognition, One shot learning, Siamese network, Triplet loss]**
+
+&#10230; [Verificação / reconhecimento facial, Aprendizado de um tiro, Rede siamesa, Perda tripla]
+
+<br>
+
+
+**10. [Neural style transfer, Activation, Style matrix, Style/content cost function]**
+
+&#10230; [Transferência de estilo neural, Ativação, Matriz de estilo, Função de custo de estilo/conteúdo]
+
+<br>
+
+
+**11. [Computational trick architectures, Generative Adversarial Net, ResNet, Inception Network]**
+
+&#10230; [Arquiteturas de truques computacionais, Rede Adversarial Generativa, ResNet, Rede de Iniciação]
```

Comment:
English is sometimes trick for portuguese speakers hahaha. What is the best: "Arquiteturas de truques computacionais" or "Truques computacionais de arquiteturas", or would they mean the same?

Reply:
I thought that the first one ("Arquiteturas de truques computacionais") was the correct translation... but there is room for both meanings I guess